web/controllers/user_controller.ex

```diff
@@ -0,0 +1,16 @@
+defmodule MovieWagerApi.UserController do
+  use MovieWagerApi.Web, :controller
+
+  alias MovieWagerApi.{Repo, User, UserSerializer}
+
+  def show(conn, %{"id" => identifier}) do
```

Comment:
Yes, we will get zombie records by default in Ember as it will add these to the store. However, we can update our serializer on the Ember side for this particular model to use. I'd rather make the front end follow this convention than the API adapt to returning single records from an index route.